mantle/build

```diff
@@ -27,6 +27,13 @@ ldflags="-X ${REPO_PATH}/version.Version=${version}"
 host_build() {
     local cmd=$1; shift
     echo "Building $cmd"
+
+    # Disable CGO if on "riscv64"
+	# https://github.com/golang/go/issues/72840
+    local cgo_override="${CGO_ENABLED:-}"
+    [ "${ARCH}" == "riscv64" ] && cgo_override=0
+
+    CGO_ENABLED=${cgo_override} \
```

Comment:
It's not clear from quickly looking at the golang docs what `CGO_ENABLED= go build ...` will do. I think it'd be easier to review if we just only set the env on riscv64.

Reply:
updated